service/src/main/java/com/codeforcommunity/processor/TeamsProcessorImpl.java

```diff
@@ -377,4 +384,15 @@ public void transferOwnership(
       throw new WrongTeamRoleException(teamId, TeamRole.LEADER);
     }
   }
+
+  @Override
+  public List<TeamDataResponse> getTeams() {
+    Result<TeamsRecord> teamsRecordResult = db.selectFrom(TEAMS).where(TEAMS.DELETED_AT.isNull()).fetch();
+    return teamsRecordResult.map(teamsRecord -> new TeamDataResponse(teamsRecord.getId(),
```

Comment:
Did you check that this works? I think usually there's some type of collect statement after the .map

Reply:
The map method in the `Result<>` interface returns a `Result<>` and that class extends `List` so this all works out. https://www.jooq.org/javadoc/latest/org.jooq/org/jooq/Result.html